qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -281,20 +239,11 @@ def stored_account() -> Dict[str, str]:
         }
 
     def active_account(self) -> Optional[Dict[str, str]]:
-        """List the IBM Q Experience v2 account currently in the session.
+        """List the IBM Q Experience account currently in the session.
 
         Returns:
             dict: information about the account currently in the session.
-
-        Raises:
-            IBMQAccountError: if an IBM Q Experience v1 account is already in
-                use.
         """
-        if self._v1_provider.active_accounts():
-            raise IBMQAccountError(
-                'IBM Q Experience v1 accounts are enabled. Please use '
-                'IBMQ.active_accounts() to retrieve information about them.')
-
         if not self._credentials:
             # Return None instead of raising, for compatibility with the
             # previous active_accounts() behavior.
```

Comment:
It would not be a bad option - however, in this case compatibility is still an issue, albeit in a slightly different form: now we need to maintain compatibility with "the previous active_account()" (ie. without the `s`, as in the 0.3 `.active_account()` method) behavior.

Being conservative and trying not introduce any changes (other than removing deprecated functions) in this round, I'll tweak the docstring slightly to better reflect it.